Aspen-Discovery/aspen-dev-box-image
# feat: Allow building of jar files from command line

This commit allows you to build any of the jar files for aspen from the command line. also includes fuzzy finding of the jar file to rebuild. the --all or -a flag will build all of the jar files (that are able to be built this way. marcMergeUtility, palace_project_export and rbdigital_export cannot be built currently in this way due to their non standard file construction.
Code located in the adb-cli repo.

to test: 
check the adb jarbuild command for the cli and check that the built jar files work as expected.

use adb jarbuild -a to build all of the jar files that are currently possible
